Sources/DVR/SessionDataTask.swift

```diff
@@ -1,6 +1,12 @@
 import Foundation
 
 final class SessionDataTask: URLSessionDataTask {
+    
+    enum TaskError: Error {
```

Comment:
additionally I replaced almost all fatalErrors with callbacks and errors as a safer way to indicate the problem